### DISCUSSION TITLE
Would this be the right tool if I want to enforce valid SQL on output?

### DISCUSSION THREAD

**Discussion by kgourgou** [outlines-dev/outlines#420] [Q&A] [answered]
Valid as in "no SQL grammar errors", not valid as in "this will run on my database when I run it".

**Comment by rlouf** [comment, marked as answer]
This will alllow you to make sure that SQL syntax is valid: https://github.com/outlines-dev/outlines/pull/391

**Reply by kgourgou** [reply]
Thank you!
